query-engine/query-engine-node-api/src/logger.rs

```diff
@@ -24,11 +15,7 @@ use tracing_subscriber::{
 };
 
 pub(crate) struct Logger {
-    // We need the original parameters in case we move to tracing
-    log_queries: bool,
-    log_level: LevelFilter,
-    log_callback: ThreadsafeFunction<String>,
-    __dispatcher: RwLock<Dispatch>,
+    __dispatcher: Dispatch,
```

Comment:
Sorry last thing then this should be good to go. Can you rename this back to `dispatcher`. I know its meant to be private with the `__` but that doesn't seem to be the convention https://rust-lang.github.io/api-guidelines/naming.html